client/release.go

```diff
@@ -21,7 +21,7 @@ func (c *HTTPClient) ListReleases(appID string) ([]releases.AppReleaseInfo, erro
 // CreateRelease adds a release to an app.
 func (c *HTTPClient) CreateRelease(appID string, opts *ReleaseOptions) (*releases.AppReleaseInfo, error) {
 	path := fmt.Sprintf("/v1/app/%s/release", appID)
-	body := &releases.Body{
+	body := &releases.BodyCreateRelease{
```

Comment:
I renamed a couple of these where `Body` was actually something like `Body5` in the generated mono-package. There are still some `BodyXX` lying around but they are unused for now